src/jdk.jfr/share/classes/jdk/jfr/internal/MetadataRepository.java

```diff
@@ -205,7 +205,6 @@ private EventConfiguration makeConfiguration(Class<? extends jdk.internal.event.
                     nativeEventTypes.remove(n);
                     nativeControls.remove(n);
                     TypeLibrary.removeType(nativeType.getId());
-                    pEventType.setAnnotations(nativeType.getAnnotationElements());
```

Comment:
This looks like accidental removal. Is it?

Reply:
It's by design. When an event is defined in both metadata.xml and as a JDK event, we want the event metadata to be taken from Java. I don't remember exactly what the problem was in this case. Perhaps @RemoveFields was ignored, so there was a field mismatch.